src/gui/search/searchwidget.h

```diff
@@ -82,6 +82,7 @@ private slots:
 
     void fillCatCombobox();
     void fillPluginComboBox();
+    void displayPage();
```

Comment:
IMO, selectActiveView (or selectActivePage) is better name for this method.
Personally I don't like using redundant parameters.